### PR TITLE
chore: bring CHANGELOG up to date after 2.0.2 release.

### PR DESCRIPTION
**Pull request by danxmoran**
I looked through the changes from #20095 and added lines here for things that weren't already covered by a line. PR links point at the original merges to master.

- [ ] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [ ] Rebased/mergeable
- [ ] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Feature flagged (if modified API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,23 +2,44 @@
...
+1. [20012](https://github.com/influxdata/influxdb/pull/20012): Validate input paths to `influxd upgrade` up-front.
+1. [20015](https://github.com/influxdata/influxdb/pull/20015): Add same site strict flag to session cookie.
 1. [20017](https://github.com/influxdata/influxdb/pull/20017): Don't include duplicates for SHOW DATABASES
 1. [20064](https://github.com/influxdata/influxdb/pull/20064): Ensure Flux reads across all shards.
 1. [20047](https://github.com/influxdata/influxdb/pull/20047): Allow scraper to ignore insecure certificates on a target. Thanks @cmackenzie1!
 1. [20076](https://github.com/influxdata/influxdb/pull/20076): Remove internal `influxd upgrade` subcommands from help text.
 1. [20074](https://github.com/influxdata/influxdb/pull/20074): Use default DBRP mapping on V1 write when no RP is specified.
 1. [20091](https://github.com/influxdata/influxdb/pull/20091): Make the DBRP http API match the swagger spec.
 
+### Refactoring & Build
```
Comment (danxmoran) [inline CHANGELOG.md:35]: What do you think about adding a section like this for changes that are mainly for the dev teams? Not sold on the title for the section, but I think it makes sense to separate them out if we want to include them in the curated log.

Comment (danxmoran) [reply CHANGELOG.md:35]: cc @timhallinflux and @psteinbachs on this question, too

Comment (gshif) [reply CHANGELOG.md:35]: We do not want expose build/refactoring changes to our users. At this point CHANGELOG has only changes in behavior (features) or/and bug fixes changes. I would not want to change this.

Comment (danxmoran) [reply CHANGELOG.md:35]: :+1: will remove it

**Review by gshif — APPROVED** [review]

**Pull request merged**